packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.types.ts

```diff
@@ -1,12 +1,22 @@
-import { ITheme } from '../../Styling';
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
 
 export interface IVerticalDividerProps {
   /**
+   * @deprecated Use styles instead.
    * Optional function to generate the class names for the divider for custom styling
    */
   getClassNames?: (theme: ITheme) => IVerticalDividerClassNames;
+  theme?: ITheme;
+  styles?: IStyleFunctionOrObject<IVerticalDividerPropsStyles, IVerticalDividerStyles>;
```

Comment:
Every newly added API (😊 and the existing ones if you are already doing work in this area) needs to be documented... meaning all props and the interfaces themselves need to have a description. The interfaces also need to have the `{@docCategory)` like here:
https://github.com/OfficeDev/office-ui-fabric-react/blob/dc8f7bcba448f00378905026f4a74f1187882a1d/packages/office-ui-fabric-react/src/components/Shimmer/Shimmer.types.ts#L5-L7